pretix_servicefees/views.py

```diff
@@ -5,23 +5,33 @@
 from pretix.base.forms import SettingsForm
 from pretix.base.models import Event
 from pretix.control.views.event import EventSettingsViewMixin, EventSettingsFormView
+from pretix.helpers.money import change_decimal_field
 
 
 class ServiceFeeSettingsForm(SettingsForm):
-    service_fee_abs = forms.DecimalField(label=_('Service fee'))
+    service_fee_abs = forms.DecimalField(label=_('Fixed fee per order'))
     service_fee_percent = forms.DecimalField(
-        label=_('Service fee (%)'),
+        label=_('Variable fee per order'),
         help_text=_('Percentage of the order total. Note that this percentage will currently only '
                     'be calculated on the summed price of sold tickets, not on other fees like e.'
                     'g. shipping fees, if there are any.')
     )
-    service_fee_abs_resellers = forms.DecimalField(label=_('Service fee with resellers'))
+    service_fee_per_ticket = forms.DecimalField(
+        label=_('Fixed fee per ticket'),
+        help_text=_('This fee will be added for each ticket sold, except for free items and addons.')
+    )
+
+    service_fee_abs_resellers = forms.DecimalField(label=_('Fixed fee per order'))
     service_fee_percent_resellers = forms.DecimalField(
-        label=_('Service fee with resellers (%)'),
+        label=_('Variable fee per order'),
```

Comment:
I think "variable" sounds wrong, shall we say "Percentual fee" maybe?

Reply:
Yeah, I was looking for a better translation as well. It looks like there's no English word for the opposite of a "fixed fee". Let's settle with "Percentual fee". :+1: